benchmarks/orc/tuplex_output/.gitignore

```diff
@@ -0,0 +1 @@
+out*
```

Comment:
I think the whole tuplex_output/ folder should be ignored. Would edit the file in benchmarks/orc/.gitgignore therefore.